bf_opencore/tests/test_custom_fields.py

```
@@ -7,15 +7,16 @@
 import json
 import pytest
 from django.db import IntegrityError
-import blueflow.models as bf_mod
-from .test_custom_field_names import cleandb, cfield
+from django.db.transaction import TransactionManagementError
+from bf_opencore import models
+
 
 def test_custom_field_api(cleandb, auth_client):
```

Ruff (0.15.1)

[warning] Unused function argument: `cleandb`

(ARG001)